source/agora/network/NetworkManager.d

```diff
@@ -230,8 +236,9 @@ public class NetworkManager
                     ledger.getBlockHeight() + 1,
                     blocks => blocks.all!(block =>
```

Comment:
We should think about fixing this in the future. It does not short-circuit.

Also there will be many `isNominating is null` checks in a tight loop here.

Reply:
We can think about fixing this in the future. It's somewhat unrelated to this PR.